Content.Client/CrewManifest/UI/CrewManifestSection.cs

```diff
@@ -35,21 +36,26 @@ public CrewManifestSection(
 
         foreach (var entry in entries)
         {
-            var name = new RichTextLabel()
+            // SS220-QoL copy name from manifest button-Begin
+            var name = new CopyableRichTextLabel()
             {
                 HorizontalExpand = true,
+                Text = entry.Name,
             };
-            name.SetMessage(entry.Name);
+            // SS220-QoL copy name from manifest button-End
```

Comment:
Либо как в предложении ниже и дальше аналогично, либо сделать API `CopyableRichTextLabel` аналогичным RichTextLabel в рамках этого кода и поменять только строчки, в которой вызывается `new RichTextLabel()`.
```suggestion
            // SS220-QoL copy name from manifest button-Begin
            //var name = new RichTextLabel()
            //{
            //    HorizontalExpand = true,
            //};
            //name.SetMessage(entry.Name);
            
            var name = new CopyableRichTextLabel()
            {
                HorizontalExpand = true,
                Text = entry.Name,
            };
            // SS220-QoL copy name from manifest button-End
```